### PR TITLE
OTE_SDK. Increase code coverage. 

### PR DESCRIPTION
**Pull request by dvkruchinin**
Tests:
ote_sdk/ote_sdk/tests/entities/test_annotation.py

Task: CVS-70204

### CODE REVIEW

**Comment by dvkruchinin** [comment]
Hi @LeonidBeynenson,
please review the PR.

**Comment by dvkruchinin** [comment]
run ote_sdk tests

**Comment by dvkruchinin** [comment]
run ote_sdk tests

**Comment by dvkruchinin** [comment]
run ote_sdk tests

**Comment by dvkruchinin** [comment]
run ote_sdk tests

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+from ote_sdk.entities.shapes.rectangle import Rectangle
+from ote_sdk.tests.constants.ote_sdk_components import OteSdkComponent
+from ote_sdk.tests.constants.requirements import Requirements
+
+
+@pytest.mark.components(OteSdkComponent.OTE_SDK)
+class TestAnnotation:
+
+    rectangle = Rectangle(x1=0.5, x2=1.0, y1=0.0, y2=0.5)
+    labels = []  # type: List[ScoredLabel]
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_annotation.py:41]: why not to use typing directly here and get rid of `# type: List[ScoredLabel]`

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+    annotation = Annotation(shape=rectangle, labels=labels)
+
+    car = LabelEntity(
+        id=123456789,  # type: ignore
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_annotation.py:45]: What do you ignore here, why not to use `id=ID(123456789)`

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py (hunk 1 above)
Comment (dvkruchinin) [reply ote_sdk/ote_sdk/tests/entities/test_annotation.py]: Fixed

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py (hunk 2 above)
Comment (dvkruchinin) [reply ote_sdk/ote_sdk/tests/entities/test_annotation.py]: Fixed

**Comment by dvkruchinin** [comment]
run ote_sdk tests

**Comment by dvkruchinin** [comment]
run ote_sdk tests

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py [hunk 3]
```diff
@@ -0,0 +1,612 @@
...
+        <b>Steps</b>
+        1. Create Annotation instances
+        2. Set another values
+        3. Check changed values
+        """
+
+        annotation = self.annotation
+        ellipse = Ellipse(x1=0.5, y1=0.1, x2=0.8, y2=0.3)
+        annotation.shape = ellipse
+        annotation.id = 123456789
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_annotation.py]: ID(...)

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py [hunk 4]
```diff
@@ -0,0 +1,612 @@
...
+        <b>Steps</b>
+        1. Create AnnotationSceneEntity instances
+        2. Set another values
+        3. Check changed values
+        """
+
+        annotation_scene_entity = self.annotation_scene_entity
+
+        creation_date = self.creation_date
+        annotation_scene_entity.id = 123456789
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_annotation.py]: ID(...)

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py [hunk 5]
```diff
@@ -0,0 +1,612 @@
...
+        annotation_scene_entity.kind = AnnotationSceneKind.PREDICTION
+        annotation_scene_entity.editor_name = "editor"
+        annotation_scene_entity.creation_date = creation_date
+        annotation_scene_entity.annotations = self.annotation
+
+        assert annotation_scene_entity.id == 123456789
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_annotation.py]: ID(...)

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py [hunk 6]
```diff
@@ -0,0 +1,612 @@
...
+        <b>Steps</b>
+        1. Create AnnotationSceneEntity instances
+        2. Check returning value of get_label_ids method
+        """
+
+        annotation_scene_entity = self.annotation_scene_entity
+
+        assert annotation_scene_entity.get_label_ids() == {ID()}
+
+        bus = LabelEntity(id=123456789, name="bus", domain=Domain.DETECTION)
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_annotation.py]: ID(...)

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

**Comment by dvkruchinin** [comment]
run ote_sdk tests

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py (hunk 3 above)
Comment (dvkruchinin) [reply ote_sdk/ote_sdk/tests/entities/test_annotation.py]: Done

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py (hunk 4 above)
Comment (dvkruchinin) [reply ote_sdk/ote_sdk/tests/entities/test_annotation.py]: Done

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py (hunk 5 above)
Comment (dvkruchinin) [reply ote_sdk/ote_sdk/tests/entities/test_annotation.py]: Done

File: ote_sdk/ote_sdk/tests/entities/test_annotation.py (hunk 6 above)
Comment (dvkruchinin) [reply ote_sdk/ote_sdk/tests/entities/test_annotation.py]: Done

**Pull request merged**